### PR TITLE
Add new ZEIT ONLINE AMP theme

### PR DESCRIPTION
**Pull request by mstoltenburg**

### CODE REVIEW

**Comment by nistormihai** [comment]
LGTM but it has conflicts

File: plugins/embed/gui-themes/themes/zeit-amp/item/source/soundcloud.dust [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{>"themeBase/item/source/base"/}
+{<itemClass}soundcloud{/itemClass}
+{<contentCore}
+	{+contentExternalContainer}{/contentExternalContainer}
+	{+contentAttr}{/contentAttr}
+{/contentCore}
+{<contentExternal}
+	<amp-soundcloud height="166" layout="fixed-height" data-trackid="{Meta.id}"></amp-soundcloud>
```
Comment (nistormihai) [inline plugins/embed/gui-themes/themes/zeit-amp/item/source/soundcloud.dust]: this is a custom amp tag, where is the script that include it?

File: plugins/embed/gui-themes/themes/zeit-amp/item/source/youtube.dust [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+{<externalInfo}
+	{+externalInfoTitleContainer}{/externalInfoTitleContainer}
+{/externalInfo}
+
+{<externalInfoTitle}{Meta.title|s}{/externalInfoTitle}
+{<externalInfoTitleSrc}//youtube.com/watch?v={Meta.id}{/externalInfoTitleSrc}
+
+{<externalYoutubeVideo}
+<amp-youtube data-videoid="{Meta.id}" width="480" height="270" layout="responsive"></amp-youtube>
```
Comment (nistormihai) [inline plugins/embed/gui-themes/themes/zeit-amp/item/source/youtube.dust]: the same question with the soundcloud

File: plugins/embed/gui-themes/themes/zeit-solo/item/base.dust [hunk 3]
```diff
@@ -3,26 +3,16 @@
 {! Attention! Opening tag in headerContainer … !}
 {<headerContainer}
 	<div class="liveblog-post">
-		<div class="post-header">
```
Comment (nistormihai) [inline plugins/embed/gui-themes/themes/zeit-solo/item/base.dust]: this are changes on the themes, should also be propagated to the html generation? or when in production

File: plugins/embed/gui-themes/themes/zeit-amp/liveblog.css [hunk 4]
```diff
@@ -0,0 +1,138 @@
+#livedesk-root { font-family: "TabletGothic", "Helvetica Neue", Helvetica, Arial, FreeSans, sans-serif; font-size: 1rem; line-height: 1.5; margin: 0 auto; max-width: 61.25em; }
```
Comment (mstoltenburg) [inline plugins/embed/gui-themes/themes/zeit-amp/liveblog.css]: For AMP, all CSS must be inside a `<style amp-custom>` tag in the head of the document. I just develop the code here and then include the output in our CMS output.

File: plugins/embed/gui-themes/themes/zeit-amp/item/source/soundcloud.dust (hunk 1 above)
Comment (mstoltenburg) [inline plugins/embed/gui-themes/themes/zeit-amp/item/source/soundcloud.dust]: In the head of the document (created at www.zeit.de). We are just embedding the liveblog HTML inside our documents using [ESI](https://en.wikipedia.org/wiki/Edge_Side_Includes). All CSS and corresponding scripts will get referenced in the template head from our backend.

File: plugins/embed/gui-themes/themes/zeit-solo/item/base.dust (hunk 3 above)
Comment (mstoltenburg) [inline plugins/embed/gui-themes/themes/zeit-solo/item/base.dust]: I just deleted this unused Element plus the corresponding CSS `.post-header { display: none; }`. I'm not sure if I understand the question right?

**Pull request merged**